events/beam-robotics.html

```diff
@@ -0,0 +1,154 @@
+    </div>
+</div>
+
+<hr class="divider" />
+
+<div class="secondary-top-section">
+    <p class="title text-2x no-margin">
+        <b>Lessons</b>
+    </p>
+</div>
+
+<div class="event-section">
+    <div class="grid-col">
+      <img class="img-responsive" src="{{site.baseurl}}/img/beam-robotics/monkey.svg" alt="a monkey from a game you'll code in our week 1 lesson, sitting under a bananna tree"/>
+    </div>
+    <div>
+      <h2 class="title text-1.5x">
+        <a class="tla-link" href="#" target="_blank" rel="noopener noreferrer">Week 1: Intro to Coding</a>  
+      </h2>
+      <p>
+        How do people make games and websites and all that cool stuff? Learn the basics of <b>coding</b> with mBlock and block code! By the end of this lesson, you will learn how to make <b> <a class="tla-link" href="https://planet.mblock.cc/project/423512" target="_blank" rel= "noopener noreferrer" >this spicy game</b> </a>. 
+    </p>
+      <div class="button-container">
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fab fa-youtube"></span> Video</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-book"></span> Student Guide</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-chalkboard-teacher"></span> Instructor Guide</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="event-section event-section-reverse">
+    <div class="grid-col">
+      <img class="img-responsive" src="{{site.baseurl}}/img/beam-robotics/beetle.svg" alt="a beetle and leaf from a game you'll code in our week 2 lesson"/>
+    </div>
+    <div>
+      <h2 class="title text-1.5x">
+        <a class="tla-link" href="#" target="_blank" rel="noopener noreferrer">Week 2: Robot Coding Basics</a> 
+      </h2>
+      <p>
+        Explore more with mBlock and learn how to make a bug move just like how a robot would. By the end of this lesson we will have completed <b><a class="tla-link" href="https://planet.mblock.cc/project/445974" target="_blank" rel= "noopener noreferrer" >our second game</a></b>! 
+      </p>
+      <div class="button-container">
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fab fa-youtube"></span> Video</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-book"></span> Student Guide</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-chalkboard-teacher"></span> Instructor Guide</a>
+      </div>
```

Comment:
since we don't have these available yet, can we leave these out? ill bop you when we have the info!